hollow/src/main/java/com/netflix/hollow/api/consumer/fs/HollowFilesystemBlobRetriever.java

```diff
@@ -223,12 +226,17 @@ public InputStream getInputStream() throws IOException {
         @Override
         public InputStream getInputStream() throws IOException {
 
-            Path tempPath = path.resolveSibling(path.getName(path.getNameCount()-1) + "-" + UUID.randomUUID().toString());
+            Path tempPath = path.resolveSibling(PARTIAL_DOWNLOAD + path.getName(path.getNameCount()-1));
+
             try(
-                    InputStream is = remoteBlob.getInputStream();
-                    OutputStream os = Files.newOutputStream(tempPath)
+                InputStream is = remoteBlob.getInputStream();
+                OutputStream os = Files.newOutputStream(tempPath, StandardOpenOption.CREATE, StandardOpenOption.APPEND);
             ) {
-                byte buf[] = new byte[4096];
+                // resume unfinished downloads
+                File outFile = new File(tempPath.toAbsolutePath().toString());
+                is.skip(outFile.length());
```

Comment:
I suppose this is better than we have currently, but I'm guessing this results in effectively the same behavior from a stream consumption perspective, i.e. we have to read the bytes (even if we drop them)?